sdk/core/src/dto/shared.dto.ts

```diff
@@ -106,6 +106,18 @@ export class SdkOptions {
 
   @IsOptional()
   issueSignupCredential?: boolean
+
+  @IsBoolean()
+  @IsOptional()
+  queryBloomVault?: boolean
+
+  @IsString()
+  @IsOptional()
+  userPoolId?: string
```

Comment:
minor: Shouldn't these (userPoolId & clientId) be under a cognito object? We may have other clientId in the future